rabbitmq-ballerina/tests/unit_tests.bal

```diff
@@ -66,7 +69,7 @@ public function testProducer() {
     Client? channelObj = rabbitmqChannel;
     if (channelObj is Client) {
         string message = "Hello from Ballerina";
-        Error? producerResult = channelObj->basicPublish(message.toBytes(), QUEUE);
+        Error? producerResult = channelObj->publishMessage({ content: message.toBytes(), routingKey: QUEUE });
```

Comment:
We can simply use 
```suggestion
        check channelObj->publishMessage({ content: message.toBytes(), routingKey: QUEUE });
```
with the function return error, right? Can get rid of the if condition.